test/e2e/load_test.go

```diff
@@ -138,16 +134,13 @@ var _ = Describe("Podman load", func() {
 
 		result := podmanTest.Podman([]string{"load", "-i", podmanTest.TempDir})
 		result.WaitWithDefaultTimeout()
-		Expect(result).Should(Exit(125))
-
-		errMsg := fmt.Sprintf("remote client supports archives only but %q is a directory", podmanTest.TempDir)
-		Expect(result.ErrorToString()).To(ContainSubstring(errMsg))
+		Expect(result).Should(ExitWithError(125, fmt.Sprintf("remote client supports archives only but %q is a directory", podmanTest.TempDir)))
 	})
 
 	It("podman load bogus file", func() {
 		save := podmanTest.Podman([]string{"load", "-i", "foobar.tar"})
 		save.WaitWithDefaultTimeout()
-		Expect(save).To(ExitWithError())
+		Expect(save).To(ExitWithError(125, "stat foobar.tar: no such file or directory"))
```

Comment:
FYI all (most?) of these `stat <name>: ...` will conflict with https://github.com/containers/podman/pull/22334 and need to be changed to faccessat

Reply:
Thanks for the heads-up. I've set to draft, to prevent this from merging. (Not that I expect it to pass remote/rootless/runc/etc tests anyway. I'm just trying to find out what the snags are).